app/lib/linters/crunch_api.rb

```diff
@@ -0,0 +1,97 @@
+require 'rest-client'
+require 'base64'
+module Linters
+
+  # A class called Crunch
+  class CrunchApi
+
+    attr_accessor :base_url
+
+    def initialize()
+      @base_url = "https://platform.42crunch.com"
+    end
+
+    # Lints an unploaded file with 42crunch.
+    #
+    # @param file [File] the file to be linted
+    # @return [String] the raw JSON response
+    def lint_to_json(file)
+
+      # Initialize local variables
+      create_response = String.new
+      created_api_id = String.new
+      report_response = String.new
+      report_data = String.new
+      decoded_report = String.new
+
+      # Post the file to 42Crunch according to their API v1 "Create an API (from file)" spec
+      # https://www.postman.com/get-42crunch/workspace/42crunch-api/request/13761657-335bcb36-c68f-43a1-823d-741b62b55bc6
+      begin
+        create_response += RestClient.post( @base_url + "/api/v1/apis",
```

Comment:
The main danger in building a URL by simple string concatenation is that one of the elements will have a backslash where it is not expected. For example here, if `@base_url` is set as `http://example.com/` the resultant url will be `http://example.com//api/v1/apis`

If you use `File.join` or `URI.join` leading and trailing backslashes will be removed from the elements and there will be less chance that someone will cause an error.

So I'd suggest:
```ruby
create_response += RestClient.post( URI.join(@base_url, "/api/v1/apis"),
```